app/services/import_empty_indices.rb

```diff
@@ -10,7 +10,7 @@ def call(options = {})
 
         if search['hits']['total'] == 0
           Rails.logger.info("Importing #{model_class} as its index is empty")
-          model_class.importer_class.new.import_and_if_possible_purge_old unless dry_run
+          ImportWorker.perform_async(model_class.importer_class.name) unless dry_run
```

Comment:
because I need to pass a simple data type to `perform_async`, since it gets serialized into redis and retrieved later when the worker is actually run.